### PR TITLE
Updated package.json and CHANGELOG for 3.4.0 release

### PR DESCRIPTION
**Pull request by Blackbaud-ErikaMcVey**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+# 3.4.0 (2019-09-24)
+
+- Added the single file attachment component. [#60](https://github.com/blackbaud/skyux-forms/pull/60)
+
+- Added the character count component. [#61](https://github.com/blackbaud/skyux-forms/pull/61)
+
+- Fixed the file drop component trash icon. [#62](https://github.com/blackbaud/skyux-forms/pull/62)
+
+- Added CONTRIBUTING.md file. [#64](https://github.com/blackbaud/skyux-forms/pull/64)
```
Comment (Blackbaud-AlexKingman) [inline CHANGELOG.md]: The `Added` portions should be grouped together. Could we move this up with the others?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,13 @@
+# 3.4.0 (2019-09-24)
+
+- Added the single file attachment component. [#60](https://github.com/blackbaud/skyux-forms/pull/60)
+
```
Comment (blackbaud-johnly) [inline CHANGELOG.md]: We usually don't leave space between lines in changelogs. Is there a reason to do it here?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,10 @@
+# 3.4.0 (2019-09-24)
+
+- Added the single file attachment component. [#60](https://github.com/blackbaud/skyux-forms/pull/60)
+- Added the character count component. [#61](https://github.com/blackbaud/skyux-forms/pull/61)
+- Added a CONTRIBUTING.md file. [#64](https://github.com/blackbaud/skyux-forms/pull/64)
+- Fixed the file drop component's trash icon. [#62](https://github.com/blackbaud/skyux-forms/pull/62)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md]: Is this correct? Looking at the PR, it looks like the changes are to the files for the file item component, not the file drop component.

File: CHANGELOG.md (hunk 2 above)
Comment (blackbaud-johnly) [reply CHANGELOG.md]: I went ahead and deleted the extra lines while making a couple other small doc tweaks.

File: CHANGELOG.md (hunk 3 above)
Comment (Blackbaud-ErikaMcVey) [reply CHANGELOG.md]: ah you're correct, it is the file item. I'll fix.

**Pull request merged**